src/test/java/com/documents/WithdrawalDocumentRequestServiceTest.java

```diff
@@ -0,0 +1,196 @@
+
+        WithdrawalDocumentRequest withdrawalDocumentRequestAfterSave = new WithdrawalDocumentRequest();
+
+        setup(withdrawalDocumentRequestAfterSave);
+
+        when(withdrawalDocumentRequestRepository.save(any(WithdrawalDocumentRequest.class))).thenReturn(withdrawalDocumentRequestAfterSave);
+
+        WithdrawalDocumentRequest savedWithdrawalDocumentRequest = withdrawalDocumentRequestServiceImpl.save(withdrawalDocumentRequestAfterSave);
+
+        assertEquals(Long.valueOf(3), savedWithdrawalDocumentRequest.getId());
+        assertEquals("DocName", savedWithdrawalDocumentRequest.getDocumentName());
+        assertEquals(Long.valueOf(34), savedWithdrawalDocumentRequest.getStudentId());
+        assertEquals("1234", savedWithdrawalDocumentRequest.getSeries());
+        assertEquals(Long.valueOf(12345), savedWithdrawalDocumentRequest.getNumberIDCard());
+        assertEquals(Long.valueOf(2), savedWithdrawalDocumentRequest.getStudyYear());
+        assertEquals(Long.valueOf(2), savedWithdrawalDocumentRequest.getUniversityYear());
+        assertEquals("IP", savedWithdrawalDocumentRequest.getTypeOfCourses());
+        assertEquals("reason", savedWithdrawalDocumentRequest.getApproveReason());
+        assertEquals("01.06.2017", savedWithdrawalDocumentRequest.getDate());
+        assertEquals("signature", savedWithdrawalDocumentRequest.getSignature());
+        assertEquals("1234", savedWithdrawalDocumentRequest.getSeriesBAC());
+        assertEquals(Long.valueOf(1234), savedWithdrawalDocumentRequest.getNumberBAC());
+        assertEquals(Long.valueOf(1234), savedWithdrawalDocumentRequest.getNumberSheet());
+    }
+
+    @Test
+    public void behavioural_withdrawalDocumentRequest_findById_should_return_true() throws Exception {
+
+        WithdrawalDocumentRequest withdrawalDocumentRequestToSave = new WithdrawalDocumentRequest();
+
+        when(withdrawalDocumentRequestRepository.findOne(any(long.class))).thenReturn(withdrawalDocumentRequestToSave);
+
+        withdrawalDocumentRequestToSave = withdrawalDocumentRequestServiceImpl.findById((long) 123);
+
+        Assert.assertNotNull(withdrawalDocumentRequestToSave);
+    }
+
+    @Test
+    public void behavioural_withdrawalDocumentRequest_deleteById_should_return_true() throws Exception {
+
+        WithdrawalDocumentRequest withdrawalDocumentRequest = new WithdrawalDocumentRequest();
+
+        withdrawalDocumentRequest.setId((long) 3);
+
+        withdrawalDocumentRequestServiceImpl.delete(withdrawalDocumentRequest.getId());
+
+        WithdrawalDocumentRequest findedWithdrawalDocumentRegistrationForm = withdrawalDocumentRequestServiceImpl.findById((long) 3);
+
+        assertNull(findedWithdrawalDocumentRegistrationForm);
+
+    }
+
+    @Test
+    public void functionality_withdrawalDocumentRequest_deleteById_should_return_delete_withdrawalDocument() throws Exception {
```

Comment:
please rename this (.....should_delete_withdrawlDocument) - erase that "delete" :D